packages/node-opentelemetry/package.json

```diff
@@ -67,6 +67,6 @@
     "pino": "^8.14.1"
   },
   "peerDependencies": {
-    "@sentry/node": "7.x"
+    "@sentry/node": "^7.0.0 || ^8.0.0"
```

Comment:
Are the changes below because of 8 backwards compatible? Will they play nice with both 7 and 8?

Reply:
yap, the code should now support both v7 and v8